python/pipeline/utils/eye_tracking.py

```diff
@@ -1,14 +1,10 @@
 from collections import defaultdict
-
-import matplotlib
+import matplotlib.pyplot as plt
 import pandas as pd
+import numpy as np
 
 from ..exceptions import PipelineException
 
-matplotlib.use('agg')
```

Comment:
I think it's fine. Your solution to the matplotlib problem should solve that as well.